src/main/java/ledger/user_interface/ui_controllers/MainPageController.java

```diff
@@ -141,4 +153,46 @@ private void createAccountPopup() {
         Scene scene = new Scene(accountController);
         this.createModal(scene, "Add Account");
     }
+
+    /**
+     * Exports the database file to the chosen directory.
+     */
+    private void exportData() {
+        JFileChooser chooser = new JFileChooser();
```

Comment:
Why are we using Swing components here? Wouldn't it be better to stay consistent with the JavaFX file chooser? The Swing one works sloppily and looks outdated...